app/services/katello/pulp/repository/yum.rb

```diff
@@ -34,8 +34,7 @@ def generate_distributors
           options = {
             protected: true,
             id: yum_dist_id,
-            auto_publish: true,
-            checksum_type: repo.saved_checksum_type || root.checksum_type
+            auto_publish: true
```

Comment:
we still want checksum_type to be set if its a pulp_master, right?

Reply:
Updated to set checksum_type for pulp_master and rely on scratch pad on capsule.